### ISSUE TITLE
improvements for ios 7.1.2+

### ISSUE THREAD

**Issue by rimgal** [reupen/ipod_manager#1]
Hi, reu. i_m was a great useful addon for foobar, very want to see improvements for ios 7.1.2 (iphone 4 final firmware) and 9xx. ty

**Comment by reupen** [comment]
Indeed. Not impossible, I just have very little time to work on this at the moment :(

**Comment by kpcenti** [comment]
Is there any chance on this being updated? This is like the only iPod syncing that actually does transcoding easily. Unfortunately I updated my iPod touch and it doesn't work with it any more
